package.json

```
@@ -153,7 +155,7 @@
     "is-plain-object": "^5.0.0",
     "known-css-properties": "^0.27.0",
     "mathml-tag-names": "^2.1.3",
-    "meow": "^9.0.0",
+    "meow": "^10.1.0",
```

[suggest] This is a minor suggestion (also `package-lock.json`), but you can ignore it.

```suggestion
    "meow": "^10.1.5",
```